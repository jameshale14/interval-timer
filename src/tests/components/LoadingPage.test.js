import React from 'react'
import { cleanup, render } from 'react-testing-library'
import LoadingPage from '../../components/LoadingPage'

afterEach(cleanup)

test('should render LoadingPage correctly', () => {
  const { container } = render(<LoadingPage />)
  expect(container).toMatchSnapshot()
})