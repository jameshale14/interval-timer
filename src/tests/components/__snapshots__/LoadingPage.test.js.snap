// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render LoadingPage correctly 1`] = `
<div>
  <div
    class="loader"
  >
    <img
      alt="Loading Animation"
      class="loader__image"
      src="images/loader.gif"
    />
  </div>
</div>
`;
